apps/system/test/marionette/shareactivityapp/manifest.webapp

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Share Activity",
+  "description": "An app which displays the details of a received share activity.",
+  "type": "certified",
+  "launch_path": "/index.html",
+  "developer": {
+    "name": "The Gaia Team",
+    "url": "https://github.com/mozilla-b2g/gaia"
+  },
+  "activities": {
+    "share": {
+      "filters": {
+        "type": ["audio/*", "image/*", "url",
+                  "video/*", "application/*", "text/vcard"]
```

Comment:
nit, one too many spaces before the start of the line (assuming github formats correctly)